tools/prost-build/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+# Copyright 2020 Shift Crypto AG
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+[package]
+name = "prost-build"
+version = "0.1.0"
+authors = ["Shift Crypto AG <support@shiftcrypto.ch>"]
+edition = "2018"
+license = "Apache-2.0"
+
+[dependencies]
+clap = "3.0.0-beta.1"
```

Comment:
Why not the latest non-beta release `2.33`?

Reply:
Why non-beta? beta is working.

Reply:
stability? otherwise, why call things "beta".
git master/HEAD is also working. why not always pull from latest git all the deps then :)

Reply:
api stability is out anyway, as 2.33 -> 3.0.0 is breaking. I expect less work to upgrade to 3.0.0 from the beta than from 2.3.3. Probably no work, would be a surprise if they changed the api that breaks our 4 line use of it.

If you insist I can do it, but imho it's working okay ;)

Reply:
> I expect less work to upgrade to 3.0.0 from the beta than from 2.33

That's the explanation I expected! Makes sense. Good as is then, considering it's 4 lines like you said. :+1:

Reply:
Well, not expected really but it makes sense either way.